product_docs/docs/epas/15/epas_compat_ora_dev_guide/05_optimizer_hints/02_access_method_hints.mdx

```diff
@@ -19,7 +19,7 @@ The following hints influence how the optimizer accesses relations to create the
 
 In addition, you can use the `ALL_ROWS`, `FIRST_ROWS`, and `FIRST_ROWS(n)` hints.
 
-From version 15 onwards, `INDEX` and `NO_INDEX` hints for the partitioned table inherit the corresponding child indexes and apply in later processing.
+From version 15 onwards, `INDEX` and `NO_INDEX` hints for the partitioned table, internally  expand to include the corresponding inherited child indexes and apply in later processing.
```

Comment:
```suggestion
`INDEX` and `NO_INDEX` hints for the partitioned table internally  expand to include the corresponding inherited child indexes and apply in later processing.
```
In this case, I could see a case for keeping the "From version 15 onwards" but I don't think it is a strong case. We might want to ask Adam if he thinks we should be making these type of statements is certain scenarios.